kinds/rolemapping/role_mapping_kind.cue

```diff
@@ -0,0 +1,23 @@
+package kind
+
+name:        "RoleMapping"
+maturity:    "merged"
+description: "Public dashboard configuration"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// 0.0
+			{
+				spec: {
+					// k8s role name
+					role: string
+					// is the identity a user or team
+					kind: "team" | "user"
+					// the team/user k8s name
+					identity: string
```

Comment:
another place where identity as a single string would be nicer.  see https://github.com/grafana/grafana/pull/68047/files#r1187966228